proxies/live/apiproxy/proxies/default.xml

```diff
@@ -63,6 +73,14 @@
         <Step>
           <Name>DecodeJWT</Name>
         </Step>
+        <Step>
+          <Name>RaiseFault.WarningDecodingJWT</Name>
+          <Condition>jwt.DecodeJWT.warning != null</Condition>
+        </Step>
+        <Step>
+          <Name>RaiseFault.ErrorDecodingJWT</Name>
```

Comment:
Is this necessary anymore - don't header errors put the DecodeJWT into the error state, which is handled by the FaultRule FailedToDecode, which is handled by AssignMessage.FailedToDecode? Can we remove this step and the RaiseFault.ErrorDecodingJWT policy?